release-team/role-handbooks/communications/README.md

```diff
@@ -81,26 +81,51 @@ The release lead will drive the content for the release theme and logo.
+
+**As feature blogs are opted in and placeholder PRs are created**, assign the blogs to shadows and yourself for tracking and facilitation. The responsibility is to ensure the blog authors have the resources and information they need, including editorial and tech reviews once ready.
+
+Add a comment to each placeholder PR to add a `/hold` label and note that the PR is awaiting publication date from the Comms team. The `hold` label prevents the PR from merging until the `hold` label is removed, this will prevent a blog from accidently merging and publishing ahead of the release. [1.31 Example](https://github.com/kubernetes/website/pull/46911#issuecomment-2185024105).
 
-**As feature blogs are opted in**, assign them to shadows and yourself for tracking and facilitation. The responsibility is to ensure the blog authors have the resources and information they need, including editorial and tech reviews once ready.
+```
+/hold
+
+Pending assignment of publication date (by release comms)
 
-**Work closely with the SIG Docs Blogs team** (Communicate with them via `#sig-docs-blogs` and by attending meetings), as they are typically available for editorial reviews. Share the feature blog schedule and updates with them throughout the cycle.
+We'll also keep this held until the vX.XX release has happened
+```
+
+**Work closely with the SIG Docs Blogs team** (Communicate with them via `#sig-docs-blogs` and by attending sig-docs meetings), as they are typically available for editorial reviews. Share the feature blog schedule and updates with them throughout the cycle.
 
 For tech reviews, reach out to authors and the sponsoring SIG to organize at least one tech review per blog post.
 
+**The first feature blog** typically goes out on release day alongside or shortly after the release blog, and then are published one-at-a-time, typically at a rate of two to three posts weekly. **The Comms team establishes the publication schedule.**
+> Note that blog PRs in k/website are dated, and automation will publish future-dated entries. This enables a PR process decoupled from blog publication date.
+
 **Communicate the planned publish date to SIG Docs and the owner of the feature blog**. 
 
 > Notify the author in the PR and Slack channels: `#sig-docs-blog`, `#sig-docs`. The communications team will assist in coordinating and publishing the feature blogs on schedule.
 > See example [here](https://github.com/kubernetes/website/pull/41924), "this article is scheduled for the 21th of August". 
 
+
 **Work with SIG Contributor Experience** (connecting on `#sig-contribex` and by attending meetings), to promote the feature blogs. 
 
 The suggested deadlines to be supported by the current Comms team:
-- Deadline - PRs blog place holder: Week 09.
+- Deadline - PRs blog place holder: Week 09
```

Comment:
- Deadline - Blog opt-in: Week 06

_This is just my guess for the opt-in deadline but I think it's good enough to both leave room to KEP owners to think about it and for Comms people to reach out to all the KEPs_

Reply:
Resolved!